### ISSUE TITLE
A11y: Edit panel: Some labels and form elements are not linked

### ISSUE THREAD

**Issue by ChristofSimon** [grafana/grafana#108096]
**Steps to reproduce**:
Edit Panel - Panel Options

**Actual Result**:
The blue framed and form elements are not linked to the labels. This means that if you click on the label with the mouse, the cursor does not jump directly into the input field.

<img width="1088" height="478" alt="Image" src="https://github.com/user-attachments/assets/ddfdf4b5-7e7c-4aa2-88ae-0dff5f897e07" />

**Expected Result**
All form elements should be linked to their labels. This means that if you click on the label with the mouse, the cursor shoulf jump directly into the input field

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)
Success Criterion 3.3.2 Labels or Instructions
https://www.w3.org/TR/WCAG21/#labels-or-instructions 

**Environment**:
- Grafana version: 12.0.1
- Data source type & version:
- User OS & Browser: Microsoft Windows 11 Enterprise / Chrome 138.0.7204.101
- Others:

**Comment by samsch** [comment]
Closed as a duplicate of https://github.com/grafana/grafana/issues/106227
This is planned work!
